VariantShredding.md

```diff
@@ -125,8 +125,7 @@ If the value is an array, `value` must be null.
 The list `element` must be a required group.
 The `element` group can contain `value` and `typed_value` fields.
 The element's `value` field stores the element as Variant-encoded `binary` when the `typed_value` is not present or cannot represent it.
-The `typed_value` field may be omitted when not shredding elements as a specific type.
-When `typed_value` is omitted, `value` must be `required`.
+The `typed_value` field may be omitted when not shredding elements as a specific type. The `value` field may be omitted when shredding elements as a specific type. However, at least one of the two fields must be present.
```

Comment:
Style: each sentence should be on a separate line in this file.

I think it is fine to allow the `value` field to be omitted if it is entirely `null`. I'd support this change.